Provided cores: 16
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	gem_2_wig
	1

rule gem_2_wig:
    input: S_cerevisiae_gem_index.gem, S_cerevisiae_gem_index_40.mappability
    output: wigs/S_cerevisiae_gem_index_40.wig
    jobid: 0
    wildcards: prefix=S_cerevisiae_gem_index, kmer=40

gem-2-wig -I S_cerevisiae_gem_index.gem -i S_cerevisiae_gem_index_40.mappability -o wigs/S_cerevisiae_gem_index_40
Error in rule gem_2_wig:
    jobid: 0
    output: wigs/S_cerevisiae_gem_index_40.wig

RuleException:
CalledProcessError in line 47 of /panfs/cmb-panasas2/skchoudh/github_projects/calculate_mappability/Snakefile:
Command 'gem-2-wig -I S_cerevisiae_gem_index.gem -i S_cerevisiae_gem_index_40.mappability -o wigs/S_cerevisiae_gem_index_40' returned non-zero exit status 1.
  File "/panfs/cmb-panasas2/skchoudh/github_projects/calculate_mappability/Snakefile", line 47, in __rule_gem_2_wig
  File "/home/cmb-panasas2/skchoudh/software_frozen/anaconda27/envs/clipseq/lib/python3.6/concurrent/futures/thread.py", line 55, in run
Will exit after finishing currently running jobs.
Exiting because a job execution failed. Look above for error message
Complete log: /panfs/cmb-panasas2/skchoudh/github_projects/calculate_mappability/.snakemake/log/2017-10-18T12:55:52.383666.snakemake.log
